apps/penxle.com/src/routes/(default)/me/settings/contentfilters/MutedSpaceModal.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts">
+  import { fragment, graphql } from '$glitch';
+  import { Button, Image, Modal } from '$lib/components';
+  import { toast } from '$lib/notification';
+  import type { FormEventHandler } from 'svelte/elements';
+  import type { MeSettingsContentFiltersPage_MutedSpaceModal_user } from '$glitch';
+
+  let _user: MeSettingsContentFiltersPage_MutedSpaceModal_user;
+  export { _user as $user };
+  export let open = false;
+
+  let mutedSpaces: typeof $user.mutedSpaces;
+
+  const setSpaces = () => {
+    mutedSpaces = $user.mutedSpaces;
+  };
+
+  $: if (open) {
+    setSpaces();
+  }
+
+  $: user = fragment(
+    _user,
+    graphql(`
+      fragment MeSettingsContentFiltersPage_MutedSpaceModal_user on User {
+        id
+
+        mutedSpaces {
+          id
+          name
+          description
+          muted
+
+          icon {
+            id
+            ...Image_image
+          }
+        }
+      }
+    `),
+  );
+
+  const muteSpace = graphql(`
+    mutation MeSettingsContentFiltersPage_MuteSpace_Mutation($input: MuteSpaceInput!) {
+      muteSpace(input: $input) {
+        id
+        muted
+      }
+    }
+  `);
+
+  const unmuteSpace = graphql(`
+    mutation MeSettingsContentFiltersPage_UnmuteSpace_Mutation($input: UnmuteSpaceInput!) {
+      unmuteSpace(input: $input) {
+        id
+        muted
+      }
+    }
+  `);
+
+  const onInput: FormEventHandler<HTMLInputElement> = (event) => {
+    const { value } = event.currentTarget;
+
+    mutedSpaces = $user.mutedSpaces.filter((space) => space.name.includes(value));
+  };
+</script>
+
+<Modal size="md" bind:open>
+  <svelte:fragment slot="title">숨긴 스페이스</svelte:fragment>
+
+  <form class="relative h-11.5 w-full mb-4" on:submit|preventDefault>
+    <input
+      class="rounded-2.5 h-11.5 w-full bg-primary py-1.75 pr-3.5 pl-11 w-full border border-bg-primary transition focus-within:border-tertiary!"
+      type="text"
+      on:input={onInput}
+    />
+    <div class="absolute inset-y-0 left-3.5 flex center text-secondary h-100%">
+      <span class="i-lc-search square-5 transition" />
+    </div>
+  </form>
+
+  <ul class="space-y-4">
+    {#each mutedSpaces as space (space.id)}
```

Comment:
P2: 검색 결과 좌우 공백 제거 및 쿼리 결과를 불변적으로 참조합시다. 

```suggestion
    query = value.trim()
  };
</script>

<Modal size="md" bind:open>
  <svelte:fragment slot="title">숨긴 스페이스</svelte:fragment>

  <form class="relative h-11.5 w-full mb-4" on:submit|preventDefault>
    <input
      class="rounded-2.5 h-11.5 w-full bg-primary py-1.75 pr-3.5 pl-11 w-full border border-bg-primary transition focus-within:border-tertiary!"
      type="text"
      on:input={onInput}
    />
    <div class="absolute inset-y-0 left-3.5 flex center text-secondary h-100%">
      <span class="i-lc-search square-5 transition" />
    </div>
  </form>

  <ul class="space-y-4">
    {#each mutedSpaces.filter(space => space.name.includes(query)) as space (space.id)}
```